packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -449,6 +449,115 @@ export const setupRDSClusterAndData = async (config: RDSConfig, queries?: string
   };
 };
 
+export const clearRDSClusterData = async (clusterInfo: ClusterInfo, region: string): Promise<void> => {
+  const client = new RDSDataClient({ region });
+
+  // Get all table names
+  const tableQuery = `
+    SELECT table_name 
+    FROM information_schema.tables 
+    WHERE table_schema = 'public'
+      AND table_type = 'BASE TABLE'
+      AND table_catalog = '${clusterInfo.dbName}'
+      AND table_name LIKE 'e2e_test_%';
+  `;
+  const tableQueryInput: ExecuteStatementCommandInput = {
+    resourceArn: clusterInfo.clusterArn,
+    secretArn: clusterInfo.secretArn,
+    sql: tableQuery,
+    database: clusterInfo.dbName,
+  };
+
+  const tableQueryCommand = new ExecuteStatementCommand(tableQueryInput);
+  let tableQueryResponse;
+  try {
+    tableQueryResponse = await client.send(tableQueryCommand);
+  } catch (err) {
+    console.log(err);
+  }
+
+  const tables = tableQueryResponse?.records?.map((record) => record[0]?.stringValue) || [];
+
+  // Truncate each table
+  for (const table of tables) {
+    const truncateQuery = `TRUNCATE TABLE ${table} CASCADE;`;
+    const truncateQueryInput: ExecuteStatementCommandInput = {
+      resourceArn: clusterInfo.clusterArn,
+      secretArn: clusterInfo.secretArn,
+      sql: truncateQuery,
+      database: clusterInfo.dbName,
+    };
+
+    const truncateQueryCommand = new ExecuteStatementCommand(truncateQueryInput);
+    try {
+      await client.send(truncateQueryCommand);
+      console.log(`Table truncated: ${table}`);
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  console.log(`[Postgres] Database [${clusterInfo.dbName}] - data cleared and all tables truncated`);
+};
+
+export const clearRDSInstanceData = async (config: RDSConfig, endpoint: string, port: number): Promise<void> => {
```

Comment:
nit: "instance" isn't the interesting characteristic of the thing we're clearing; it's that we're accessing via direct DFB connection. Suggest renaming to reflect that, "clearTestDataUsingDirectConnection" or similar.

Reply:
Function name changed